ui/src/js/profile/ToolsPage.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import Boostrap from 'bootstrap';
+import axios from 'axios';
+
+import Tools from './tools/Tools';
+
+export default class ToolsPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+      },
+      pageCount: 0
+    };
+  }
+
+  callNextPage() {
+    var newPage = this.state.pageCount + 1;
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
+  callPreviousPage() {
+    var newPage = this.state.pageCount - 1;
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
+  updateLanguage(value) {
+    this.refs['navbar'].updateLanguage(value);
+  }
+
+  render() {
+    var that = this;
+    var toolsPart;
+    var page = this.state.pageCount;
+
+    if (this.state.user.editAllowed) {
+      toolsPart = <Tools/>;
+    } else {
+      toolsPart = '';
+    }
+
+    toolsPart = <Tools/>;
```

Comment:
This overrides always the set value from line 40 to 44. Maybe this is wrong placed or the lines from 40 to 44 are not necessary.

Reply:
Removed unnecessary code